tools/generate-helpers/pg-table-bindings/resource_type_test.go

```diff
@@ -46,7 +46,7 @@ func TestGetResourceType(t *testing.T) {
 		{typ: &storage.ComplianceOperatorCheckResultV2{}, resourceType: directlyScoped},
 		{typ: &storage.ComplianceOperatorClusterScanConfigStatus{}, resourceType: directlyScoped},
 		{typ: &storage.ComplianceOperatorProfile{}, resourceType: globallyScoped},
-		{typ: &storage.ComplianceOperatorProfileV2{}, resourceType: indirectlyScoped},
+		{typ: &storage.ComplianceOperatorProfileV2{}, resourceType: directlyScoped},
```

Comment:
Wondering what `directlyScoped` differs from `indirectlyScoped`

Reply:
directly scoped means the cluster is in the table and we can directly use that to limit access based on user cluster permissions.  Indirectly scoped means it has to go to an intermediate table to determine that.